### PR TITLE
test(ssz): add runner for `ssz_static` spec tests

### PR DESCRIPTION
**Pull request by MegaRedHand**

### CODE REVIEW

File: test/spec/runners/ssz_static.ex [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    assert {:ok, decompressed} = :snappyer.decompress(compressed)
+
+    expected =
+      YamlElixir.read_from_file!(case_dir <> "/value.yaml")
+      |> Stream.map(&parse_yaml/1)
+      |> Map.new()
+
+    expected_root = YamlElixir.read_from_file!(case_dir <> "/roots.yaml")
+
+    assert_ssz(testcase, decompressed, expected, expected_root)
```
Comment (mpaulucci) [inline test/spec/runners/ssz_static.ex:33]: Can we include the Elixir struct here somewhere?

like: decompressed = %Checkpoint{}

Comment (mpaulucci) [reply test/spec/runners/ssz_static.ex:33]: So that we ensure that the Elixir structs are respecting the schema.

Comment (MegaRedHand) [reply test/spec/runners/ssz_static.ex:33]: I don't know if this is what you mean, but I intend to add the Elixir-side structs and integrate that with the SSZ lib and the test runner in another PR.

Comment (mpaulucci) [reply test/spec/runners/ssz_static.ex:33]: sounds good

**Review by mpaulucci — APPROVED** [review]

**Pull request merged**